example.cjs

```diff
@@ -1,4 +1,4 @@
-const fetchHAR = require('.').default;
+const fetchHAR = require('.');
```

Comment:
Won't this change break things for CJS users? Or will it also work from `.default`?

Reply:
nah the `default` is no longer required — this is likely going to be a breaking change! the `default` thing was actually a bit of a code smell since we probably weren't exporting types properly

Reply:
Worth calling out in the changelog (or in the commit message)?

Reply:
yep will do in the GitHub release!